### DISCUSSION TITLE
Who's Hiring (July 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#52399] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #52398.

Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by allang** [comment]
Folks | Freelance Front-End Engineer | Remote | allang@gmail.com

**Who We Are**
We are a team of 3 building an AI-powered customer support product. Our customer-facing product is a NextJS app. We are a few weeks away from shipping our first version.

**The Role**
We need a NextJS developer on a freelance basis. This includes tasks such as billing, some admin tools, and ongoing feature development as we grow.

**Technologies**
- NextJS
- Typescript
- Prisma
- React-form
- Zod
- Nextauth

Please mention your experience with these technologies and whether you’ve used them in a project.

**Skills**
- 4+ years of experience working with React and JavaScript, with a preference for proficiency in Typescript.
- Experience with Postgres and Prisma is preferred.
- Experience working on a NextJS app with database operations in the services layer, HTTP logic and service calls in controllers, and reusable logic in a middleware layer.
- Must have a good grasp of English.

If you have public GitHub work, please include it. Also, let us know if you’ve worked with Stripe before (preferred but not required).

Email allang@gmail.com to apply.

**Reply by coolidev** [reply]
Hi. I am interested in this position.
Can I connected with you directly and discuss further?

**Reply by Susmita-Dey** [reply]
Hi, I'm interested in this position. 

Here's my portfolio:- https://susmitadey.me

**Comment by jessepinho** [comment]
## Senior full-stack engineer
Hey all! Jesse Pinho here 👋🏻

I’m a senior full-stack engineer and technical advisor with deep experience in both backend and frontend — from Rails and Node (NestJS/Express) on the backend to React, NextJS, and Gatsby on the frontend.

I have availability for the next few months for new projects. My sweet spot is working with companies who have a new web-based project to build from the ground up: building the app, writing unit tests, writing documentation for future maintainers, and conducting a handoff to ensure its ongoing success. I also enjoy joining an existing team to get project launches across the finish line on a solid technical foundation.

My resume is here: https://jessepinho.com/assets/resume.pdf and my email is [jesse@jessepinho.com](mailto:jesse@jessepinho.com)

Shoot me a message if you’d like to chat about working together! Looking forward to hearing from you 🙂

Location: Los Angeles, CA, USA
Remote: Yes
Willing to relocate: No
Technologies: React, NextJS, TypeScript, Jest, Gatsby, Webpack, NodeJS, NestJS, Ruby on Rails, React Native
Resume/CV: https://jessepinho.com/assets/resume.pdf
Website: https://jessepinho.com/ 
Email: [jesse@jessepinho.com](mailto:jesse@jessepinho.com)

**Reply by moinulmoin** [reply]
This is “Who’s hiring” page, not [”Who wants to be hired”](https://github.com/vercel/next.js/discussions/52398)

**Comment by creteurlouis** [comment]
**Acoustic Wells| Senior Full-Stack Engineer | Remote (Remote or Boston) | Full-time or Part-time | [www.acoustic-wells.com/careers](http://www.acoustic-wells.com/careers)**

At Acoustic Wells we are a fast paced, dynamic team dedicated to experimentation and growth.
We manage key pieces of Oil & Gas infrastructure using IOT, Machine learning & Data Science.
We are rapidly expanding, and value key partners who can move with us as we grow and learn.
We’re looking for a forward-thinking Senior Front-end Engineer to help build out our different dashboards, APIs, Web applications and also work on the global architecture of our solutions.
Are you ready to hop on board an exciting, MIT born, Data Science & IoT startup? Are you ready to help drive innovation and not be a “ticket puncher”?
Step up and let’s do this!

**Who We Are**
We are a startup in the energy and IoT space, based on research done at MIT. We are developing a solution to help oil and gas producers reduce their environmental footprint and increase their field efficiency by allowing them to automatically control and monitor their assets remotely.

**Technologies**
Front:
- React (Next JS)
- Typescript
- Material UI

Back:
- Python
- FastAPI
- .NET


AWS
Postgresql

**Skills / Experience**
4+ years of experience working with React/Next.js using Typescript
3+ years of experience with .NET or Python (Flask / FastAPI or Django)
PostgreSQL
MUI

**Apply Here**: [www.acoustic-wells.com/careers](http://www.acoustic-wells.com/careers)

**Reply by BFF271** [reply]
Hi, @creteurlouis 
I'm very interested in your project.

**Reply by SuperButterfly** [reply]
Hi. Nice to meet you. I am really interested in your position.
I am a senior full stack developer with over 10 years experience. 
Please contact me. 
Thanks
My email: topworldstar0101@gmail.com

**Comment by josephskewes** [comment]
**ID8R | Front-End Engineer / Cofounder | Remote | Part Time | [joseph@id8r.com](mailto:joseph@id8r.com)**

We are looking for a front end engineer to join our bootstrapped startup (ID8R) as cofounder. We have a technical lead, but looking for someone to accelerate our progress and own the front end of our app and websites.

What we are building:

_ID8R empowers NFT collectors and artists to personalize, showcase, and enhance their digital assets (‘Canva for web3’)._

We are looking for someone with:

- An interest in crypto, web3, and NFTs
- Strong, demonstrable experience in JS, React, Next.js, HTML5, CSS, UI/UX, also, -> WebGL and/or Three.js would be highly regarded
- Consistent availability (20+ hours/week) and **ok with vesting equity as remuneration until funding/revenue supports salaries**

A little about our progress:

- We are a passionate team of 3 cofounders, geographically remote (AU, IN, DE)
- We have an MVP and we are adapting what we learn from early customers into the product
- We have some early sales revenue, but it's not recurring and we intend to introduce a subscription model soon
- We recently secured a grant (Shimmer ecosystem) and won the Solana NFT Showdown
- [Read our whitepaper](https://drive.google.com/file/d/1Bjh3K7WcXMLf2E3ty1bi_ONYfHcOJ6PC/view) to learn more about the who, what, and why we are building ID8R
- We are in the early stages of seeking pre-seed investment, but it's a tight funding environment and we want to accelerate what we are doing regardless of outcome

Open to questions in the comments or by email. I don't currently have this listed on any job sites to submit an application, but you can express interest by emailing me a short introduction and links to your GitHub and portfolio: joseph@id8r.com

**Reply by radna0** [reply]
Can I contact you via Discord?

**Reply by josephskewes** [reply]
@radna0, yes I am on Discord at: joseph_au

**Reply by Moncef-Mhz** [reply]
HI, i am interested 
Portfolio: https://portfolio-orpin-beta-50.vercel.app/

**Reply by josephskewes** [reply]
Thanks for the interest @Moncef-Mhz, best if you can apply here so I can capture what's needed to move to a video call: https://cryptojobslist.com/jobs/lead-front-end-engineer-cofounder-equity-id8r-remote

**Comment by spider-yamet** [comment]
Dear Hiring Manager,

I am writing to express my interest in the NextJS Developer position on a freelance basis at your esteemed company. I am excited about the opportunity to contribute my skills in NextJS, Typescript, Prisma, React-form, Zod, and Nextauth to support your billing, admin tools, and ongoing feature development as the company grows.

Allow me to introduce myself and highlight my qualifications:

Proficient in NextJS, I have hands-on experience developing scalable and performant web applications, delivering excellent user experiences.
Skilled in Typescript, I leverage its static typing and advanced features to write robust, maintainable code, ensuring high-quality deliverables.
Experienced with Prisma, I am well-versed in database management and building efficient data models for seamless integration with applications.
Knowledgeable in React-form and Zod, I have successfully implemented dynamic and interactive forms with validation, ensuring data integrity and user-friendly experiences.
Proficient in Nextauth, I can seamlessly integrate authentication and user management functionalities into applications, enhancing security and user engagement.
Strong understanding of modern web development practices, including RESTful APIs, state management, and responsive design.
Self-motivated and adaptable, I thrive in dynamic environments, delivering results efficiently and meeting project deadlines.
Effective communicator, I collaborate seamlessly with cross-functional teams to understand requirements and deliver solutions that meet business objectives.
I am enthusiastic about the prospect of contributing my expertise to your company's growth and success. As a freelance NextJS developer, I am committed to delivering high-quality code, contributing to feature development, and ensuring smooth user experiences.

Please find my attached resume, detailing my relevant work experience and notable projects. I am available for an interview to discuss how my skills align with your company's needs and how I can add value to your team.

Thank you for considering my application. I look forward to the opportunity of working with you and contributing to the success of your projects.

Sincerely,

John
needmefordev@gmail.com

**Comment by spider-yamet** [comment]
Dear Joseph,

Just sent friend request with my account to you
Hope you will confirm it as soon as possible

Best regards,
John Smith

пон, 17. јул 2023. у 08:11 Joseph ***@***.***> је написао/ла:

**Comment by JamesRobertWiseman** [comment]
OneSpark | Senior Front-End Engineer | Remote (South Africa) | Full-time | [Link](https://info.onespark.co.za/careers/senior-front-end-engineer/)

### Who we are
Traditional life insurance is old, and kind of boring. But at OneSpark, we’re different. We’re building digital insurance products that make people’s lives better through easy-to-use digital journeys connected to clever AI algorithms, delivering world-first solutions to add true value to customers' lives. These innovations have resulted in global backing, partnerships with giants like Google, multiple insurance accolades, and despite our young age, billions of rands in cover already sold.

### Who we are looking for
You must have experience designing and building large and complex (yet maintainable) systems, websites and apps, and you should be able to do so in about one-third of the time that most competent people think possible.

You should have a BSc, Msc or PhD in Computer Science, Engineering or the equivalent. Top-notch communication and management skills are essential.

### What you’ll be doing
Much of what is being built is a world first, so expect a diverse range of tasks requiring development across various UI layers from mail design and implementation to front end website dev including CSS, HTML and JavaScript (Node.js, Typescript, React). A large part of the build will be greenfields work, so come prepared to work hard and challenge yourself.

### What you'll need
- ***Must reside in South Africa.***
- A minimum of 4+ years of relevant experience.
- Excellent HTML5, CSS and TypeScript (React) knowledge.
- Experience with UX/UI/Product design patterns.
- Incredibly detailed oriented.
- Collaborate with backend engineers in ensuring technical feasibility.
- Experience working in an agile development environment.
- Next.js, MJML or other mail markup language knowledge a plus.

### What we offer
- Global exposure, with work focusing on multiple continents
- Hybrid work environment, with flexibility between at home and the office.
- Unlimited annual leave.
- Flexible work hours.
- Opportunity to work with highly skilled, intense, and interesting coworkers who share a desire to change the world, starting with insurance.

### Interested?
Reach out to me directly or send us an email with your CV and include the subject line `Senior Front End Engineer // OneSpark`, to [careers.tech@onespark.co.za](mailto:careers.tech@onespark.co.za)

**Comment by sean-11x** [comment]
Hiring designers to join us in building the biggest AI SaaS company in Europe, email me sean@11x.ai

**Reply by Susmita-Dey** [reply]
Hey @sean-11x Are you hiring developers?

**Comment by jeroenransijn** [comment]
Good Entry (just me atm) | Front-end developer | Remote | Contract work | $500-1000 (just a homepage) | [Apply here](https://airtable.com/appQwZuKSLyRXSUF1/shrQ85wXMlLGIwQXp)

**Description**
Are you interested in helping me with a product homepage for a SaaS product I have been building for the last couple of years? I might have more work after that if things go well. 

I am a senior product designer and engineer and currently lead design systems at Airtable with over 10 years of experience. This is a collaborative project to work alongside me. As there’s much more to do for this project, I am trying to outsource part of the work.
﻿

**Project**

- I need help building a responsive homepage that is partially build: https://www.goodentrycrypto.com/
- I mostly want to focus on the Figma designs and hand it off to you, these are still work in progress: https://www.dropbox.com/scl/fi/rf4ibvb8d3tppzvhsafkk/Good-Entry-marketing-homepage-Aug-3-2023.png?rlkey=x597tum3f899ig8wngt1jc72e&dl=0
﻿

Timezone
- Flexible
﻿

Work style

- High collaboration with me
- Zoom / video conferencing if timezone permits
- Async code reviews
﻿

Skills

- I am looking for someone that is focused primarily on front-end development and can implement designs as good or better than provided.
- Preferably you should be able to fill in the gaps for responsive layouts. We can work together and collaborate, I will be writing code and code reviewing as well.

**Duration**

- Duration and time is flexible, there’s no rush on this project.
Budget
- Budget is flexible depending on the scope and quality of work, we can break into deliverables. Happy to pay up to 50% up front.
Stack:

- React
- Next.js (Blitz.js)
- TypeScript
- Mostly custom CSS-in-JS with a `sx` prop. I am also using Mantine.dev, but for the homepage not so much. 
- GitHub
- Railway

Please share your portfolio or email in the thread, I will reach out to your email.

**Reply by radna0** [reply]
Here's my email Jeroen Ransijn: giabach.hoang@arol.dev

**Reply by moizghumann** [reply]
hit me up on my email,
moizghuman@gmail.com

**Reply by DevMiguell** [reply]
here's my email: buguel2000@gmail.com

**Reply by burkaslarry** [reply]
Hit me up on my email. 
lo.wailun5@gmail.com

And my website 
https://larrywebsite.vercel.app

**Reply by Susmita-Dey** [reply]
Portfolio:- https://susmitadey.me

Here's my email:- susmitadey475@gmail.com

I'm interested to work on the project. Hit me up on my email. Looking forward to chat with you.

**Reply by VektorTech** [reply]
Portfolio: https://kesutherland.com/
Email: kennysutherland@protonmail.com

**Reply by coolidev** [reply]
Hi. @jeroenransijn 
> Here's my email: gloproman0211@gmail.com
> I am ready for the video call.
> Please feel free to schedule time to call at noon in workdays.
> Thanks.

**Reply by hugorydel** [reply]
Hey! This is Hugo <;
Email: hugorydel@gmail.com

**Reply by silverchef36** [reply]
Hi Jeroen, thanks for your posting.
I am seasoned frontend engineer with 9 years of professional experience.
Currently, I am self-employed looking for new opportunities in area, thus, fully free at the moment.
Being extremely interested in your project, I am ready to dive into it now.
I also have an idea to develop further to enhance our project with you, since you have expertise in design.
Here's my portfolio and linkedin:
https://jeffrey-childers.me
https://linkedin.com/in/jeffrey-childers-tech
FYI, my email address: royalmediaus@gmail.com
Please reach me out to discuss in detail and proceed asap.
Regards,
Jeffrey

**Comment by VanessaCrid** [comment]
[Arcade Software](https://www.arcade.software/) | Full Stack Engineer | San Francisco, CA (hybrid 2x/week) | Full-time | $175k-$200k base, .1-.3% equity | [Apply Here!](https://jobs.ashbyhq.com/arcade/0d5c297e-fd18-4afc-9a7b-aaadc4bffe81)

Arcade is a seed-stage start up that builds software for product marketers - our first product is a demo-making tool. Our mission is to empower the Product Led Growth economy by making it super simple to create beautiful demos that tell the story of their product. 

We are currently looking for Full Stack Product Engineers who embrace product-led development by understanding and empathizing with our user's challenges and business needs. You'll join a highly collaborative and fun team to build delightful products that value the user experience!

On the front end, our stack is heavily Jamstack-based with Next.js, Typescript, as well as a Chrome Extension. On the back end, we use GCP (Firestore, Cloud Functions, PubSub, IdP, and a few other services) and AWS (Aurora). Arcades are embedded on our users' site, which means we strive for high-performing code and we must maintain high performance to make that work well. We have some really interesting challenges around video processing and AI on our roadmap. 

If you're excited about moving fast, love working directly with users/customers and want an amazing team culture, this is the perfect fit for you! Please feel free to apply directly or share your resume/portfolio to me at vanessa@arcade.software!
